src/main/java/org/cryptomator/ui/unlock/UnlockInvalidMountPointController.java

```diff
@@ -21,43 +22,75 @@ public class UnlockInvalidMountPointController implements FxController {
 
 	private final Stage window;
 	private final Vault vault;
-	private final AtomicReference<Throwable> unlockException;
 	private final FxApplicationWindows appWindows;
 	private final ResourceBundle resourceBundle;
 
+	private final ExceptionType exceptionType;
+	private final String exceptionMessage;
+
 	public FormattedLabel dialogDescription;
 
 	@Inject
 	UnlockInvalidMountPointController(@UnlockWindow Stage window, @UnlockWindow Vault vault, @UnlockWindow AtomicReference<Throwable> unlockException, FxApplicationWindows appWindows, ResourceBundle resourceBundle) {
 		this.window = window;
 		this.vault = vault;
-		this.unlockException = unlockException;
 		this.appWindows = appWindows;
 		this.resourceBundle = resourceBundle;
+
+		var exc = unlockException.get();
+		this.exceptionType = getExceptionType(exc);
+		this.exceptionMessage = exc.getMessage();
 	}
 
 	@FXML
 	public void initialize() {
-		var e = unlockException.get();
-		var translationKey = switch (e) {
-			case MountPointNotSupportedException x -> "unlock.error.customPath.description.notSupported";
-			case MountPointNotExistsException x -> "unlock.error.customPath.description.notExists";
-			case MountPointInUseException x -> "unlock.error.customPath.description.inUse";
-			default -> "unlock.error.customPath.description.generic";
-		};
-		dialogDescription.setFormat(resourceBundle.getString(translationKey));
-		dialogDescription.setArg1(e.getMessage());
+		dialogDescription.setFormat(resourceBundle.getString(exceptionType.translationKey));
+		dialogDescription.setArg1(exceptionMessage);
 	}
 
 	@FXML
 	public void close() {
 		window.close();
 	}
 
+	public boolean isShowPreferences() {
+		return exceptionType.showPreferences;
+	}
+
```

Comment:
Can you move this to the end of the class, see for example https://github.com/cryptomator/cryptomator/blob/2c984ad4059ff680e6c3ed0701094c8e23d74cf3/src/main/java/org/cryptomator/ui/preferences/AboutController.java#L40-L48

Reply:
Resolved by 70117605e52b3a2bbb588a356e88a4007ded30a2